spec/factories/equipment_reservations.rb

```diff
@@ -0,0 +1,6 @@
+FactoryBot.define do
+  factory :equipment_reservation do
+    reservation { nil }
+    equipment { nil }
```

Comment:
This is the only change I think could be improved. Instead of having it nil it could be like this:

`reservation {association :reservation} `
`equipment {association :equipment} `